x-pack/plugins/security_solution/public/timelines/components/row_renderers_browser/row_renderers_browser.tsx

```diff
@@ -105,6 +109,7 @@ const RowRenderersBrowserComponent = ({
       <EuiCheckbox
         id={item.id}
         onChange={handleNameClick(item)}
+        className={`${!excludedRowRendererIds.includes(item.id) ? 'isSelected' : 'isNotSelected'}`}
```

Comment:
Can we pull this out into a `useMemo`?

Reply:
Problem is that this expression is already inside a useCallback and needs an `item` parameter. 

It could be extracted to `useCallback` instead of `useMemo` but that would not help much.

Reply:
Yea, I didn't expand to see the context so yea, doesn't make any sense to do that. Thanks!